.gitignore

```diff
@@ -10,3 +10,153 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# our ignored files
```

Comment:
Have you pulled all changes? As I'm quite sure that I specified `intellij` for `gi`. Also, below you have number of entries for `idea` dir.